packages/custom_lint/lib/src/analyzer_plugin/analyzer_plugin.dart

```diff
@@ -69,13 +69,14 @@ class CustomLintPlugin extends ServerPlugin {
       final changedFiles = {...next.keys, ...?previous?.keys};
 
       for (final changedFile in changedFiles) {
-        if (previous?[changedFile] != next[changedFile]) {
+        if (previous?[changedFile] != next[changedFile] || _invalidateLints) {
```

Comment:
Do you remember in which scenario this new condition is necessary?

From my checks, the `if` is correctly reached without it. But maybe I missed something

Reply:
No, I don't remember, it could have been due to a change in riverpod or how the provider are structured.